download/alpha.html

```diff
@@ -11,17 +11,21 @@ <h5>The latest Alpha MonoDevelop release is: <strong>{{ site.data.alpharelease.v
     <p>Please choose your operating system to view the available packages. Source code is available on <a href="https://github.com/mono/monodevelop">GitHub</a> (viewing only) or as a <a href="https://download.mono-project.com/sources/monodevelop/">Tarball</a>.</p>
 
     <dl id="mono-download" class="tabs horizontal" data-tab data-options="deep_linking: true; scroll_to_content: false">
-      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;Mac OS X</a></dd>
+      <dd class="active"><a href="#download-mac"><i class="fa fa-apple"></i>&nbsp;macOS</a></dd>
       <dd><a href="#download-lin"><i class="fa fa-linux"></i>&nbsp;Linux</a></dd>
       <dd><a href="#download-win"><i class="fa fa-windows"></i>&nbsp;Windows</a></dd>
     </dl>
 
     <div class="tabs-content" id="mono-download-panel">
       <div class="panel content active" id="download-mac" style="padding: 20px 10px 10px 10px">
-        <h3>Visual Studio for Mac is available as a Mac Disk Image (.dmg)</h3>
+        <h3>MonoDevelop for macOS is available from source only</h3>
+        <p>Please refer to the <a href="/developers/building-monodevelop/">building guide</a> for more information about how to install and configure your MonoDevelop.</p>
+        <hr />
+        <h3>Visual Studio for Mac is available to install</h3>
         <div>
-          <a href="{{ site.data.alpharelease.monodevelop_mac_url }}" class="button radius"><i class="fa fa-download"></i> Download Visual Studio for Mac</a>
-          <p>Supported on Mac OS X 10.11 and later.</p>
+          <p>Visual Studio for Mac builds on top of MonoDevelop, bringing in open sourced internals from Visual Studio and many new extensions to support new workloads.</p>
+          <a href="https://www.visualstudio.com/vs/visual-studio-mac/" class="button radius">Install Visual Studio for Mac</a>
```

Comment:
So this means we won't give a direct link to the .dmg anymore? given this is the alpha page I assume it means people will need to download the stable version and then update to alpha via the updater? If yes we should probably add a sentence which explains this.